app/api/sessions.py

```diff
@@ -86,7 +86,7 @@ def query(self, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        query_ = self.session.query(Session)
+        query_ = self.session.query(Session).filter(Session.state == "accepted")
```

Comment:
deleted_at=None

Reply:
@iamareebjamal I tested it, in this particular case as well, it doesn't return rows with `deleted_at != null`

Reply:
How does it do that automatically?

Reply:
@iamareebjamal I will debug which part of the code does that, but I tested it, it was not returning deleted sessions.

Reply:
@iamareebjamal This is filtered by flask-rest-jsonapi by default. 

https://github.com/fossasia/flask-rest-jsonapi/blob/4b10b48e443a839bbff369c769198ede34fdbff3/flask_rest_jsonapi/data_layers/alchemy.py#L94

Reply:
OK :+1: